tests/distributed_multinode_test.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 Google LLC
```

Comment:
Why not just add this to distributed_test.py under a different class?

Reply:
Sure I can do that. Just wanted to start with a separate file. Multi-node tests would have env vars like `SLURM...` and although I've added `skipIfs` to ensure that they don't interfere with other tests, I'd like to know how everyone feels here. 
@skye @hawkinsp @jakevdp

Reply:
I think for now we can keep this as a separate test, since we're still iterating on its exact content & setup, but we'll revisit this in the future

Reply:
Then let's rename this: `distributed_multiprocess_gpu_test.py`. The reason I said to merge is because distributed_test.py was created for this exact reason.